internal/clientcache/internal/cache/status.go

```diff
@@ -41,12 +41,18 @@ type AuthTokenStatus struct {
 	KeyringlessReferences int
 }
 
+type BoundaryStatus struct {
+	// The boundary address for this user
+	Address          string
```

Comment:
Just to check, this doesn't end up being serialized and sent back to the user, right?

Reply:
Ah nm, I thought it was also somewhere else and I see it below!